romana-install/roles/system/tasks/ssh_keys.yml

```diff
@@ -27,3 +27,7 @@
 - name: Add known_hosts
   template: src="known_hosts" dest="~/.ssh/known_hosts" mode=0644
   when: not kh.stat.exists
+
+- name: Copy github key
+  copy: src="{{ github_key }}" dest="~/.ssh/" mode=0600
```

Comment:
not sure if its going to make it bit harder to implement but i don't like this part, any chance we can clone repo locally and copy it to remote? or maybe just point to the local copy of repo and it copies that it itself, no branch checkout, etc, something like if core is specified use that locally, rest from romana/<repo> etc..